lib/generate-suggestions.ts

```diff
@@ -57,83 +60,89 @@ async function findComponentsInJSXFiles(params: {
 
   const promises: Promise<any>[] = [];
 
-  for (const file of files) {
-    promises.push(
```

Comment:
I didn't change any of this logic, I just stuffed it into a `handleFile` function